packages/wallets/src/components/WalletsAddMoreCarousel/WalletsAddMoreCarousel.tsx

```diff
@@ -45,11 +47,17 @@ const WalletsAddMoreCarousel = () => {
                     onMouseLeave={() => !is_mobile && setIsHovered(false)}
                 >
                     <div className='wallets-add-more__carousel-wrapper'>
-                        {data?.map(item => (
-                            <div className='wallets-add-more__card' key={item.currency}>
-                                {item.currency}
-                            </div>
-                        ))}
+                        {available_wallets?.map(item => {
+                            const { currency, is_added, landing_company_name } = item;
+                            return (
+                                <WalletsAddMoreCard
+                                    key={`${currency}${landing_company_name}`}
```

Comment:
```suggestion
                                    key={`wallets_add_more_${currency}-${landing_company_name}`}
```
to make it more specific use this key